### ISSUE TITLE
Custom widgets within VuetifyTemplate components?

### ISSUE THREAD

**Issue by nmearl** [mariobuikhuizen/ipyvuetify#29]
Is it possible to use regular ipywidget objects inside a vuetify template class? The following simple example doesn't seem to work:

```python
select = IntSlider(
    value=7,
    min=0,
    max=10,
    step=1
)

layout = v.Layout(children=[select])

class Test(v.VuetifyTemplate):
    
    items = Any(['a', 'b']).tag(sync=True)
    template = Unicode('''
            <v-layout column>
              <myselect/>
              <v-select :items="items"/>
            </v-layout>
''').tag(sync=True)
    
    
    def vue_menu_click(self, data):
        self.color = self.items[data]
        self.button_text = self.items[data]
    
    
test = Test(components={'myselect': layout})
test
```

**Comment by mariobuikhuizen** [comment]
Thanks for reporting this issue.

It is supported, but was broken before v0.1.11.

**Comment by nmearl** [comment]
Great, thanks. Should I go ahead and downgrade, or is this something that might be solved soon?

**Comment by mariobuikhuizen** [comment]
It is fixed in v0.1.11 which is the latest version.

**Comment by nmearl** [comment]
Indeed! Thank you!
